packages/validator/src/plugins/validation/2and3/semantic-validators/schema-ibm.js

```diff
@@ -241,12 +241,12 @@ function generateDescriptionWarnings(
   const hasDescription =
     schema.description && schema.description.toString().trim().length;
   if (isTopLevelSchema && !hasDescription) {
-    messages.addMessage(
-      contextPath,
-      'Schema must have a non-empty description.',
-      config.no_schema_description,
-      'no_schema_description'
-    );
+    // messages.addMessage(
```

Comment:
I wanted to remove the old no_schema_description and no_property_description rule functionality without disrupting the operation of the other rules that are also implemented in this same file, so I opted to just comment out the addition of the error messages.   Once we convert over all the rest of the rules implemented in this file, we can just remove the file altogether and be done with it :)